concepts/arrays/introduction.md

```diff
@@ -33,7 +33,7 @@ numbers;
 
 ## Methods
 
-Some of the [methods][array_methods] that are available on `Array.prototype` can be used to add or remove from the array.
+Some of the [methods][array_methods] that are available on `Array.prototype`[^0] can be used to add or remove from the array.
```

Comment:
I think the best way forward here would be to write "... that are available on every array can be used...". We have a separate concept that introduces `prototype` in detail later on.

Would be great if you could apply this fix to all the 3 files mentioned in the automated PR comment.